spec/datadog_spec.rb

```diff
@@ -24,7 +24,7 @@
 
     agent.wait_for_metrics
 
-    expect(agent.metrics.count).to eq 1
+    expect(agent.metrics.count).to eq 7
```

Comment:
After downgrading the dogstatsd-ruby version library the right expectation here is 7 not 1.